CMakeLists.txt

```diff
@@ -20,7 +20,10 @@ include(IgnCMake)
 
 #--------------------------------------
 # Set up the project
-ign_configure_project(VERSION_SUFFIX)
+ign_configure_project(
+  REPLACE_IGNITION_INCLUDE_PATH gz/cmake
+  VERSION_SUFFIX pre1
```

Comment:
are we planning to make a prerelease?

Reply:
Oh, I thought the missing VERSION_SUFFIX was a bug :X
Let me remove